spec/models/snap_application_attributes_spec.rb

```diff
@@ -53,7 +53,6 @@
         financial_accounts_mutual_funds_or_stocks: false,
         financial_accounts_four_oh_one_k_or_iras: true,
         financial_accounts_trusts: false,
-        first_member_with_disability_name: nil,
```

Comment:
removed on purpose?

Reply:
Yes, this was a duplicate and was throwing a Ruby-level warning for the duplicated key. It looks like someone else fixed this after I rebased!